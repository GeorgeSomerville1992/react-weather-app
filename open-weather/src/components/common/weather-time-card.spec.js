import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import WeatherTimeCard from './weather-time-card';

describe('<WeatherTimeCard>', () => {
  it('renders structure', () => {
    const tree = renderer.create(
      <WeatherTimeCard
        currentTemp={310}
        currentTime={198456}
      />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})