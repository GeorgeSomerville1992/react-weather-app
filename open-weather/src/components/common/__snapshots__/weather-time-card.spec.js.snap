// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WeatherTimeCard> renders structure 1`] = `
<div
  className="MuiPaper-root-2 MuiPaper-elevation1-5 MuiPaper-rounded-3 MuiCard-root-1 sc-bwzfXH zvJZ"
>
  <div
    className="MuiCardMedia-root-29"
    src={undefined}
    style={
      Object {
        "backgroundImage": "url(\\"/static/images/cards/contemplative-reptile.jpg\\")",
      }
    }
    title="Contemplative Reptile"
  />
  <div
    className="MuiCardContent-root-31"
  >
    <p
      className="MuiTypography-root-32 MuiTypography-body1-41"
    >
      7:07 AM
    </p>
    <p
      className="MuiTypography-root-32 MuiTypography-body1-41"
    >
      37
       Â°C
    </p>
  </div>
</div>
`;
